game.py

```diff
@@ -31,7 +31,20 @@
 
 
 class Player:
-    def __init__(self, health: int = 100, Stones: int = 0, Smokes: int = 0, Place: str = "Entrance", Screw: bool = False, Key: bool = False, RedKey: bool = False, GreenKey: bool = False, Firester: bool = False, Gryphon: bool = False, WoodenSword: bool = False, GoldenSword: bool = False) -> None:
+    def __init__(
+     self,
+     health: int = 100,
+     Stones: int = 0,
+     Smokes: int = 0,
+     Place: str = "Entrance",
+     Screw: bool = False,
+     Key: bool = False,
+     RedKey: bool = False,
+     GreenKey: bool = False,
+     Firester: bool = False,
+     Gryphon: bool = False,
+     WoodenSword: bool = False,
+     GoldenSword: bool = False) -> None:

```

Comment:
Why would i do this?

Reply:
PEP 8 recommends a line width limit of 70